ipypublish/preprocessors/slides_from_markdown.py

```diff
@@ -174,8 +174,20 @@ def preprocess(self, nb, resources):
         latexdoc_tags = ["code", "error", "table", "equation", "figure", "text"]
         # break up titles
         cells_in_slide = 0
-        header_levels = []
         final_cells = FinalCells(self.header_slide)
+
+        header_levels = []
+        if nb.metadata.toc is not None:
```

Comment:
`if "toc" in nb.metadata:` should pass the tests

Reply:
I have fixed this issue by adding except clause.  The tests now pass.